FW/Inc/app_typedef.h

```diff
@@ -1,11 +1,11 @@
 /*
- * Filename: bpt_mem_map.g
+ * Filename: philip_mem_map.g
```

Comment:
hmmm I think I am not very attentive.  Maybe just hardcode to app_typedef.h so in the code it is right.